extensions/examples/online-extensions.js

```diff
@@ -41,8 +41,8 @@ angular
     'extensionRegistry',
     function(extensionRegistry) {
 
-      var system_status_elem = $('<a href="http://status.openshift.com/" target="_blank" class="nav-item-iconic system-status project-action-btn">');
-      var system_status_elem_mobile = $('<div row flex class="navbar-flex-btn system-status-mobile">');
+      var system_status_elem = $('<a href="http://status.openshift.com/" target="_blank" class="nav-item-iconic system-status project-action-btn" style="display: none;">');
+      var system_status_elem_mobile = $('<div row flex class="navbar-flex-btn system-status-mobile" style="display: none;">');
```

Comment:
Oi, good Q. That was a good long time ago.  I vaguely remember eliminating it as it was extra logic not really doing much, but w/o pulling it and running it again prob cant tell you.